atom/browser/api/lib/auto-updater/auto-updater-win.js

```diff
@@ -2,70 +2,62 @@ const app = require('electron').app;
 const EventEmitter = require('events').EventEmitter;
 const url = require('url');
 const squirrelUpdate = require('./squirrel-update-win');
+const util = require('util');
 
-var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-hasProp = {}.hasOwnProperty;
+function AutoUpdater() {
+  AutoUpdater.super_.call(this);
```

Comment:
> The cleaner option is to use es2015 classes (e.g. class AutoUpdater extends EventEmitter { ... }).

Great idea, thanks so much for suggesting this! :+1: 

I will push up changes to use them instead and remove use of `util.inherits`.